README.md

```diff
@@ -2,45 +2,41 @@ This repository contains source code for the SafeTab-P disclosure
 avoidance application. SafeTab-P was used by the Census Bureau for the
 protection of individual 2020 Census responses in the tabulation and
 publication of the Detailed Demographic and Housing Characteristics
-File A (DDHC-A). A later source code release will contain the source
+File A (Detailed DHC-A). A later source code release will contain the source
 code for SafeTab-H, the application used to protect the Detailed
-Demographic and Housing Characteristics File B (DDHC-B).
+Demographic and Housing Characteristics File B (Detailed DHC-B).
 
-Using the mathematical principles of formal privacy, SafeTab-P infused
-noise into Census survey results to create *privacy-protected
-microdata* which were used by Bureau subject matter experts to
-tabulate the 2020 DDHC-A product.  SafeTab-P was built on Tumult's
-"Analytics" and "Core" platforms. both SafeTab-P and the underlying
-platforms are implemented in Python. The latest version of the
-platforms can be found at https://tmlt.dev/.
+Using the mathematical principles of differential privacy, SafeTab-P infused
+noise into 2020 Census results to create *privacy-protected
+statistics* which were used by Census Bureau subject-matter experts to
+tabulate the 2020 Detailed DHC-A product. SafeTab-P was built on Tumult Analytics, a platform for computing statistics using differential privacy. Both SafeTab-P and the underlying platform are implemented in Python. The latest version of Tumult Analytics can be found at https://tmlt.dev/.
```

Comment:
The changes seem to have made the line wrapping inconsistent here (and in a more minor way, in the rest of this file)

Reply:
Looks good in browser too. Resolving.